geoh5py/io/h5_reader.py

```diff
@@ -98,21 +98,12 @@ def fetch_attributes(
                 ][:]
 
             if "Value map" in entity["Type"].keys():
-                value_map = entity["Type"]["Value map"][:]
-                mapping = {}
-                for key, value in value_map.tolist():
-                    value = cls.str_from_utf8_bytes(value)
-
-                    mapping[key] = value
-
+                mapping = cls.fetch_value_map(file, uid)
```

Comment:
this would require its dedicated issue, PR and test, rather than being engulfed in a merge to main for a release